README.md

```diff
@@ -2,43 +2,24 @@
   <img src="logo.png">
 </p>
 
-> 📣🆕 [https://api.sorare.com/federation/graphql](https://api.sorare.com/federation/graphql) is the new Sorare API endpoint. It's a federated superset of existing API endpoints so all existing queries remain valid, just replace the URL in your application. Visit the [playground](https://api.sorare.com/federation/graphql/playground).
-
 # Sorare API
 
 At Sorare, we are committed to providing an open platform for developers to build upon.
 
 While our Cards are stored on the Ethereum blockchain (or within a [Starkware rollup](https://starkware.co/starkex/)) we support an API that provides more detailed information.
 
-The Sorare API are provided by [GraphQL](https://graphql.org/). Sorare provides a federated API for all sports. The API is hosted on [https://api.sorare.com/federation/graphql](https://api.sorare.com/federation/graphql). The documentation can be found under the Docs section of the [GraphQL playground](https://api.sorare.com/federation/graphql/playground).
+The Sorare API are provided by [GraphQL](https://graphql.org/). The API is hosted on [https://api.sorare.com/graphql](https://api.sorare.com/graphql). The documentation can be found under the Docs section of the [GraphQL playground](https://api.sorare.com/graphql/playground).
```

Comment:
this we cannot merge yet

Reply:
Today:
- /graphql and /federation/graphql are the same endpoints
- /graphql/playground redirects to /federation/graphql/playground; we might want to remove the redirect and expose /graphql/playground to target /graphql